packages/vaex-core/vaex/settings.py

```diff
@@ -102,7 +102,7 @@ class AsyncEnum(str, Enum):
 try:
     import vaex.server
     has_server = True
-except ImportError:
+except (AttributeError, ImportError):
```

Comment:
Maybe if we import vaex.server.settings instead, it should continue to work with the older vaex server